glide.lock

```diff
@@ -1,10 +1,6 @@
 hash: 6e87e4ae2a53b3118fbaa16f113df872177c284b0cb9ce7e996faa14ea1912be
 updated: 2017-05-26T12:33:16.52452318-04:00
```

Comment:
Hash/timestamp need to be updated:

```
[WARN]	Lock file may be out of date. Hash check of YAML failed. You may need to run 'update'
```

Running **glide update** should do that.